docs/user-docs/getting_started.rst

```diff
@@ -141,15 +148,30 @@ specify the feed URL as well as a comma separated list of package names you wish
 
    This command may be exited via ctrl+c without affecting the request.
 
-
-   Downloading and processing metadata.
-   [-]
+   Downloading Python metadata.
+   [==================================================] 100%
+   2 of 2 items
    ... completed
 
    Downloading and processing Python packages.
    [==================================================] 100%
-   30 of 30 items
+   718 of 718 items
    ... completed
 
 
    Task Succeeded 
+
+   Publishing Python Metadata.
+   [-]
+   ... completed
+   
+   Publishing Python Content.
+   [-]
+   ... completed
+   
+   Making files available via web.
+   [-]
+   ... completed
+   
+   
+   Task Succeeded
```

Comment:
Can you please add a section on how to sync from one pulp python repository to another pulp python repository. Thanks!